Wave/Wave/RowType.swift

```diff
@@ -13,4 +13,5 @@ public protocol RowType {
     
     func registerCells(tableView: UITableView)
     func configureCell(tableViewCell: UITableViewCell)
+    func didSelectCell(tableViewCell: UITableViewCell, tableView: UITableView, indexPath: NSIndexPath)
```

Comment:
Yeah I think we can change it to just row,  but we need the tableView because it reloads it on tapping